model/buildlogger_test.go

```diff
@@ -365,42 +364,29 @@ func TestBuildloggerAppend(t *testing.T) {
 	t.Run("AppendToBucketAndDB", func(t *testing.T) {
 		log.Setup(env)
 		require.NoError(t, log.Append(ctx, chunk1))
+		time.Sleep(time.Millisecond)
```

Comment:
Can you add a comment explaining that appends must not overlap so they must occur in different miliseconds?

Reply:
Done.